### PR TITLE
[CIVIS-9536] Add documentation for AWS CodePipelines running pipelines

### PR DESCRIPTION
**Pull request by ManuelPalenzuelaDD**
### What does this PR do? What is the motivation?

Add documentation for the running pipelines feature for AWS CodePipeline

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ further_reading:
 ## Compatibility
 
 - **Partial pipelines**: View partially retried executions
+- **Running pipelines**: View pipeline executions that are running
+    - NOTE: Running pipelines don't have git information until they have finished.
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:26]:
```suggestion
    - **Note**: Running pipelines don't have Git information until they have finished.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just requesting a minor style update

File: content/en/continuous_integration/pipelines/_index.md [hunk 2]
```diff
@@ -155,10 +155,10 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
 
 ### Supported features
 
-|  | Jenkins | GitLab | CircleCI | Buildkite | GitHub Actions | Azure Pipelines | Codefresh | TeamCity | AWS Code Pipeline | Other CI Providers |
+|  | Jenkins | GitLab | CircleCI | Buildkite | GitHub Actions | Azure Pipelines | Codefresh | TeamCity | AWS CodePipeline | Other CI Providers |
```
Comment (ManuelPalenzuelaDD) [inline content/en/continuous_integration/pipelines/_index.md:158]: The CI provider name is AWS CodePipeline

**Review by drodriguezhdez — APPROVED** [review]
LGTM

**Review by buraizu — APPROVED** [review]
LGTM; let me know when this PR is ready to merge

**Comment by ManuelPalenzuelaDD** [comment]
@buraizu Thank you! It should be ready, i forgot to tick the box 🙇

**Pull request merged**